pkg/datapath/linux/linux_defaults/mark.go

```diff
@@ -72,6 +72,11 @@ const (
 	// MagicMarkHost determines that the traffic is sourced from the local
 	// host and not from a proxy.
 	MagicMarkHost int = 0x0C00
+
+	// MagicMarkIdentity determines that the traffic carries a security
+	// identity in the skb->mark
+	MagicMarkIdentity int = 0x0F00
+
```

Comment:
Should the comment above the `const` block be updated?

Reply:
The meaning of the flag is unchanged. We use the fact that the identity is carried as proof that another Cilium endpoint is the source of the traffic.

Reply:
@borkmann has since removed that comment since it was out of date